Home_Work_5/task_26.py

```diff
@@ -0,0 +1,14 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+# *Пример:*
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8
+
+def pow(a, b):
+    if b == 0:
+        return 1
```

Comment:
Можно сократить решение на один вызов функции.
```suggestion
    if b == 1:
        return a
```

Reply:
Дз находится в презентации к уроку, как и задачи к самому занятию!
Спасибо за комментарии